dail/tpqy-response.vbs

```diff
@@ -82,6 +83,11 @@ Else
     Call navigate_to_MAXIS_screen("INFC", "SVES")
     EmWriteScreen client_SSN, 4, 68
     Call write_value_and_transmit("TPQY", 20, 70)
+
+	'checking for NON-DISCLOSURE AGREEMENT REQUIRED FOR ACCESS TO IEVS FUNCTIONS'
```

Comment:
This needs to be added to the if SSN_present = True statement.

Reply:
corrected